ql/cashflows/couponpricer.hpp

```diff
@@ -80,12 +84,31 @@ namespace QuantLib {
         Handle<OptionletVolatilityStructure> capletVol_;
     };
 
-    //! Black-formula pricer for capped/floored Ibor coupons
+    /*! Black-formula pricer for capped/floored Ibor coupons
+        References for timing adjustments
+        Black76             Hull, Options, Futures and other
+                            derivatives, 4th ed., page 550
+        BivariateLognormal  http://ssrn.com/abstract=2170721
+        The bivariate lognormal adjustment implementation is
+        still considered experimental */
     class BlackIborCouponPricer : public IborCouponPricer {
       public:
-        BlackIborCouponPricer(const Handle<OptionletVolatilityStructure>& v =
-                                        Handle<OptionletVolatilityStructure>())
-        : IborCouponPricer(v) {};
+        enum TimingAdjustment { Black76, BivariateLognormal };
+        BlackIborCouponPricer(
+            const Handle< OptionletVolatilityStructure > &v =
+                Handle< OptionletVolatilityStructure >(),
+            const TimingAdjustment timingAdjustment = Black76,
+            const Handle< Quote > correlation =
+                Handle< Quote >(boost::make_shared< SimpleQuote >(1.0)))
```

Comment:
Do you think 1 is a sensible default value for correlation when not given explicitly, or was it just to avoid an empty handle?